spec/models/skin_spec.rb

```diff
@@ -28,6 +28,10 @@
         dd {color: blue;}
         /* end comment */",
 
+      "should allow hsl(a) colours" =>
+        "ol {color: hsl(100, 100, 100);}
```

Comment:
Let's use [the right syntax for `hsl`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hsl) here: the first value can be a unitless number, but the second and third values for saturation and lightness should be percentages. The same applies to `hsla` as well.